server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationStatsIT.java

```diff
@@ -121,6 +131,10 @@ public void testSegmentReplicationStatsResponseForActiveAndCompletedOnly() throw
             completedOnlyResponse.shardSegmentReplicationStates().get(INDEX_NAME).get(0).getStage(),
             SegmentReplicationState.Stage.DONE
         );
+        assertThat(
```

Comment:
We can use `assertEquals` instead of `assertThat` here. This is applicable for other usages of `assertThat` as well